bin/reload-local.sh

```diff
@@ -219,7 +219,7 @@ then
 
   # Do we need to download a remote dump?
   # Best if we do this before droping current database.
-  if [ ${REFRESH_LOCAL_DUMP} = true ] || [ ! -f $LOCAL_FILE ]
+  if [ ${REFRESH_LOCAL_DUMP} = true ] || [ ! -f "${LOCAL_FILE}.gz" ]
```

Comment:
Instead of adding '.gz' to several places like this line I would just change the LOCAL_FILE definition:

From
```
LOCAL_FILE=${PROJECT_ROOT}tmp/${BACKUP_FILE_NAME}
```

to 


```
LOCAL_FILE=${PROJECT_ROOT}tmp/${BACKUP_FILE_NAME}.gz
```